include/deal.II/multigrid/mg_transfer.templates.h

```diff
@@ -210,6 +210,8 @@ MGLevelGlobalTransfer<VectorType>::copy_to_mg(
   MGLevelObject<VectorType> &      dst,
   const InVector &                 src) const
 {
+  Assert(sizes.size() == dof_handler.get_triangulation().n_levels(),
+         ExcMessage("MGLevelGlobalTransfer::build() has not been called!"));
```

Comment:
I think we should write 
```suggestion
         ExcMessage("MGTransfer::build() has not been called!"));
```
The problem is that `build` sits in derived classes like `MGTransferMatrixFree` or `MGTransferPrebuilt`; now we cannot give the right name, so I suggest a simple placeholder that might give an indication of what happens already. Or to extend this to say `The derived MG transfer class did not call build()`.